src/metadata.h

```diff
@@ -47,8 +47,10 @@ namespace librealsense
         META_DATA_INTEL_L500_DEPTH_CONTROL_ID   = 0x80000012,
         META_DATA_CAMERA_DEBUG_ID               = 0x800000FF,
         META_DATA_HID_IMU_REPORT_ID             = 0x80001001,
-        META_DATA_HID_CUSTOM_TEMP_REPORT_ID     = 0x80001002,
+        META_DATA_HID_CUSTOM_TEMP_REPORT_ID     = 0x80001002, 
         META_DATA_INTEL_SAFETY_ID               = 0x80000014,
+        META_DATA_INTEL_OCCUPANCY_ID            = 0x80000016,
```

Comment:
Is this something the FW team should now about or it's internal only?

Reply:
This is part of the ICD - so FW are aware of that